Cargo.toml

```diff
@@ -19,8 +19,9 @@ fix-lint        = "./scripts/fix-lint.sh"
 integration     = "./scripts/integration_test.sh"
 
 [workspace.dependencies]
-babylon-merkle        = { git = "https://github.com/babylonlabs-io/cosmos-bsn-contracts" }
-eots                  = { git = "https://github.com/babylonlabs-io/cosmos-bsn-contracts" }
+babylon-bindings = { git = "https://github.com/babylonlabs-io/bindings", branch = "f/release-0.2" }
```

Comment:
Since the custom message in Babylon is subjected to change and now we have GRPC queries, what do you think we

- use GRPC queries rather than custom queries, similar to [here](https://github.com/babylonlabs-io/rollup-bsn-contracts/blob/5c9f56e48bb58efdaf5cb08ffa2c2bf657d74766/contracts/finality/src/utils.rs#L22-L27),
- create Rust structs in this repo rather than introducing the dependency to `babylon-bindings`?

Reply:
OK then, and agree that GRPC queries are quite brittle at the current version of CosmWasm.

Reply:
Perhaps something can be done by leveraging `prost` or any protobuf crate, for deserialization. And by using the structs / defs under the `babylon-proto` package, that come from babylon and are well defined.

But the current impl, using `Bufany` and explicit field ids and types, is horrible.